File: SDKs/Aspose.Pdf_Cloud_SDK_for_CSharp/src/Com/Aspose/PDF/Model/AnnotationResponse.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;

namespace Com.Aspose.PDF.Model {
  public class AnnotationResponse {
    public Annotation Annotation { get; set; }

    public string Code { get; set; }

    public string Status { get; set; }

    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class AnnotationResponse {\n");
      sb.Append("  Annotation: ").Append(Annotation).Append("\n");
      sb.Append("  Code: ").Append(Code).Append("\n");
      sb.Append("  Status: ").Append(Status).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }
  }
  }

File: SDKs/Aspose.Pdf-Cloud-SDK-for-Android/Aspose-PDF-Cloud-SDK-for-Android/src/main/java/com/aspose/pdf/model/AnnotationResponse.java
package com.aspose.pdf.model;

import com.aspose.pdf.model.Annotation;
public class AnnotationResponse {
  private Annotation Annotation = null;
  private String Code = null;
  private String Status = null;
  /**
	 * getAnnotation
	 * Gets Annotation
	 * @return Annotation
	 */
  public Annotation getAnnotation() {
    return Annotation;
  }

	/**
	 * setAnnotation
	 * Sets Annotation
	 * @param Annotation Annotation
	 */
  public void setAnnotation(Annotation Annotation) {
    this.Annotation = Annotation;
  }

  /**
	 * getCode
	 * Gets String
	 * @return Code
	 */
  public String getCode() {
    return Code;
  }

	/**
	 * setCode
	 * Sets String
	 * @param Code String
	 */
  public void setCode(String Code) {
    this.Code = Code;
  }

  /**
	 * getStatus
	 * Gets String
	 * @return Status
	 */
  public String getStatus() {
    return Status;
  }

	/**
	 * setStatus
	 * Sets String
	 * @param Status String
	 */
  public void setStatus(String Status) {
    this.Status = Status;
  }

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class AnnotationResponse {\n");
    sb.append("  Annotation: ").append(Annotation).append("\n");
    sb.append("  Code: ").append(Code).append("\n");
    sb.append("  Status: ").append(Status).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}

